front-page.php

```diff
@@ -1,9 +1,27 @@
 <?php
 get_header();
-?>
 
+// Get splash image
+$splash_images = get_post_meta($post->ID, '_igv_home_splash_images', true);
```

Comment:
This (combined with setting the meta boxes on the home page) requires setting a manual homepage in the reading options. I don't like that as we are using front-page.php which is an override. As there is no other data to manage for this front page I think these values should come from site settings. But this is a data question where opinions vary